Description: Patch to set some compilation flags for various builds.
Origin: http://anonscm.debian.org/gitweb/?p=pkg-games/alure.git;a=tree;f=debian/patches

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,6 +88,31 @@
         SET(STATIC_CFLAGS "${STATIC_CFLAGS} -fPIC")
     ENDIF(NOT WIN32)
 
+    SET(CMAKE_C_FLAGS_RELWITHDEBINFO "-g -O2 -D_DEBUG" CACHE STRING
+        "Flags used by the compiler during Release with Debug Info builds."
+        FORCE)
+    SET(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g -O2 -D_DEBUG" CACHE STRING
+        "Flags used by the compiler during Release with Debug Info builds."
+        FORCE)
+    SET(CMAKE_C_FLAGS_MINSIZEREL "-Os -DNDEBUG" CACHE STRING
+        "Flags used by the compiler during release minsize builds."
+        FORCE)
+    SET(CMAKE_CXX_FLAGS_MINSIZEREL "-Os -DNDEBUG" CACHE STRING
+        "Flags used by the compiler during release minsize builds."
+        FORCE)
+    SET(CMAKE_C_FLAGS_RELEASE "-O3 -fomit-frame-pointer -DNDEBUG" CACHE STRING
+        "Flags used by the compiler during release builds"
+        FORCE)
+    SET(CMAKE_CXX_FLAGS_RELEASE "-O3 -fomit-frame-pointer -DNDEBUG" CACHE STRING
+        "Flags used by the compiler during release builds"
+        FORCE)
+    SET(CMAKE_C_FLAGS_DEBUG "-g3 -D_DEBUG" CACHE STRING
+        "Flags used by the compiler during debug builds."
+        FORCE)
+    SET(CMAKE_CXX_FLAGS_DEBUG "-g3 -D_DEBUG" CACHE STRING
+        "Flags used by the compiler during debug builds."
+        FORCE)
+
     CHECK_C_COMPILER_FLAG(-Wextra HAVE_WEXTRA)
     IF(HAVE_WEXTRA)
         ADD_DEFINITIONS(-Wextra)
